### PR TITLE
Update active/active documentation.

### PR DESCRIPTION
**Pull request by amy-hashi**
### What
Updated the documentation for TFE Active/Active installs to include information about sticky sessions on load balancers/app gateways. 

I also removed the cluster warning note at the top because it's been over two years since we pulled the plug on clustering.

### Why
A customer experienced an issue in an active/active deployment in Azure with SAML authentication because stick sessions were not enabled. Support was able to resolve the issue, but to prevent future support tickets, this update makes the information available for all three major clouds.

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: content/enterprise/install/automated/active-active.mdx [hunk 1]
```diff
@@ -265,6 +263,14 @@ As you scale Terraform Enterprise beyond 2 nodes, you may be adding additional s
 - [Google Cloud - Network Quotas and Limits](https://cloud.google.com/vpc/docs/quota)
 - [Google Cloud - API rate limits](https://cloud.google.com/compute/docs/api-rate-limits)
 
+Depending on your infrastructure and Terraform Enterprise configuration, you may need to configure your application gateway or load balancer for "sticky sessions." For example, an Active/Active deployment on Azure with SAML authentication requires sticky sessions be enabled to ensure the authentication with the SAML server is successful. The terminology for this varies across clouds. Please reference the documentation for your infrastructure.
```
Comment (laurapacilio) [inline content/enterprise/install/automated/active-active.mdx:266]: My only comment here is: Could we provide the briefest definition of "sticky session" for folks? Or are we assuming that everyone will be familiar with this already? Just checking!

Comment (amy-hashi) [reply content/enterprise/install/automated/active-active.mdx]: Okay, added another sentence w/a definition.

File: content/enterprise/install/automated/active-active.mdx [hunk 2]
```diff
@@ -265,6 +263,14 @@ As you scale Terraform Enterprise beyond 2 nodes, you may be adding additional s
 - [Google Cloud - Network Quotas and Limits](https://cloud.google.com/vpc/docs/quota)
 - [Google Cloud - API rate limits](https://cloud.google.com/compute/docs/api-rate-limits)
 
+Depending on your infrastructure and Terraform Enterprise configuration, you may need to configure your application gateway or load balancer for "sticky sessions." Sticky session is the common term for a setting in a load balancer or gateway device that ensures traffic is routed back to the original system that initiated a given request. For example, an Active/Active deployment on Azure with SAML authentication requires sticky sessions be enabled to ensure the authentication with the SAML server is successful. The terminology for this varies across clouds. Please reference the documentation for your infrastructure.
```
Comment (laurapacilio) [inline content/enterprise/install/automated/active-active.mdx:266]:
```suggestion
Depending on your infrastructure and Terraform Enterprise configuration, you may need to configure your application gateway or load balancer for sticky sessions. Sticky session refers to the practice of using a load balancer or gateway device to ensure traffic is routed back to the original system that initiated a request. For example, an Active/Active deployment on Azure with SAML authentication requires sticky sessions to ensure the authentication with the SAML server is successful. The terminology for this varies across clouds. Refer to the documentation for your infrastructure.
```

**Review by laurapacilio — APPROVED** [review]
A couple of nits :) if you approve, you're good to merge! 🚀

**Pull request merged**